app/views/monograph_catalog/_index_gallery_file_set_wrapper.html.erb

```diff
@@ -2,7 +2,7 @@
 <div class="document col-xs-6 col-md-3">
   <div class="thumbnail">
     <% if fs_presenter.use_riiif_for_icon? %>
-      <% icon_string = '<img src="/image-service/' + fs_presenter.id + '/full/200,/0/default.png' + fs_presenter&.browser_cache_breaker + '" >' %>
+      <% icon_string = '<img src="/image-service/' + fs_presenter.id + fs_presenter&.browser_cache_breaker + '/full/200,/0/default.png' + '">' %>
```

Comment:
I'm almost positive we're doing this on purpose since the work title is present else where and we don't want screen readers to read it twice. I'll bring this up at standup.

Reply:
People agree: we do this on purpose for the reasons I mentioned.